traverson4j-core/src/test/java/uk/co/autotrader/traverson/conversion/FastJsonResourceConverterTest.java

```diff
@@ -4,12 +4,13 @@
 import com.alibaba.fastjson.JSONObject;
 import org.apache.commons.io.IOUtils;
 import org.assertj.core.data.MapEntry;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
```

Comment:
Lets aim to reduce the dependency on hamcrest in favour for assertJ

Reply:
same reason for using hamcrest